### PR TITLE
Project Portfolio Page [Jing Zhe]

### PR DESCRIPTION
**Pull request by AJZ1995**

### CODE REVIEW

File: docs/team/jingzhe.adoc [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+== Contributions to the User Guide
+
+
+|===
+|_Given below are sections I contributed to the User Guide. They showcase my ability to write documentation targeting end-users._
+|===
+
+include::../UserGuide.adoc[tag=undoredo]
+
+include::../UserGuide.adoc[tag=dataencryption]
```
Comment (amad-person) [inline docs/team/jingzhe.adoc]: I think you're supposed to update this to what sections you've contributed to in the UG, right?

File: docs/team/jingzhe.adoc [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+== Contributions to the Developer Guide
+
+|===
+|_Given below are sections I contributed to the Developer Guide. They showcase my ability to write technical documentation and the technical depth of my contributions to the project._
+|===
+
+include::../DeveloperGuide.adoc[tag=undoredo]
```
Comment (amad-person) [inline docs/team/jingzhe.adoc]: This should be updated too.

File: docs/team/jingzhe.adoc (hunk 1 above)
Comment (AJZ1995) [reply docs/team/jingzhe.adoc]: I haven't update.

File: docs/team/jingzhe.adoc (hunk 2 above)
Comment (AJZ1995) [reply docs/team/jingzhe.adoc]: I haven't update.

**Pull request merged**